pennylane/optimize/rotosolve.py

```diff
@@ -14,78 +14,122 @@
 """Rotosolve gradient free optimizer"""
 # pylint: disable=too-many-branches,cell-var-from-loop
 
+from inspect import signature
 import numpy as np
 from scipy.optimize import brute, shgo
-from pennylane.utils import _flatten, unflatten
```

Comment:
I don't yet know why, but super nice to see these removed!

Reply:
Haha I can relate :see_no_evil: